ProyectoSemestral3/Assets/Scripts/PlayerMovement.cs

```diff
@@ -4,50 +4,24 @@
 
 public class PlayerMovement : MonoBehaviour
 {
-    public CharacterController controller;
-    private Vector3 playerVelocity;
-    private bool isitGrounded;
-    public float speed = 5;
-    public float jumpHeight = 1f;
-    public float gravity = -9.81f;
+    [SerializeField] private CharacterController _controller;
+    [SerializeField] private Transform cam;
+    [SerializeField] private float _speed = 6f;
+    [SerializeField] private float _turnTime = 0.1f;
+    float _turnVelocity;
```

Comment:
Why you did not add an access modifier and default value here? XD

Reply:
you mean for the turnvelocity?